.github/scripts/sync-merged-pr-to-wiki.sh

```
@@ -0,0 +1,155 @@
+
+## 작업내용
+
+$title
+
+$body
+
+## PR 본문 원문
+
+$body
+
+_last synced: $synced_at_
```

Shellcheck (0.11.0)

[warning] synced_at_ is referenced but not assigned (did you mean 'synced_at'?).

(SC2154)